### PR TITLE
Mapsandmatching

### PR DESCRIPTION
**Pull request by BrandonWade**
This contains the recent changes to the development branch as well as the google-maps branch.

### CODE REVIEW

File: webapp/bower.json [hunk 1]
```diff
@@ -17,6 +17,14 @@
     "angular-route": "~1.3.12",
     "bootstrap": "~3.3.2",
     "angular-mocks": "~1.3.12",
-    "angular-ui-router": "~0.2.13"
+    "angular-ui-router": "~0.2.13",
+    "angular-google-maps": "~2.0.12",
+    "angularjs-google-places": "~0.1.0"
```
Comment (kjrcda) [inline webapp/bower.json:22]: need comma after the line, otherwise cant install

File: .gitignore [hunk 2]
```diff
@@ -1,3 +1,5 @@
 *DS_STORE*
-npm-debug.log
 .idea
+**/npm-debug.log
```
Comment (kjrcda) [inline .gitignore]: why the **/ in front of the file?

**Comment by kjrcda** [comment]
heyo, checkout the PR I made instead of this one. I think figured a way to make the files changed portion wayyyyy cleaner and the merge as well. Let me know what you think.

**Comment by BrandonWade** [comment]
Closing this since the changes were merged with more maps work. See PR #45 since it incorporates the aforementioned changes and doesn't mark every file as new.

**Pull request closed without merging**